Almond/99_bottles/99_bottles.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+
+for ((i=1; i<=99; i++)); do
```

Comment:
I'd probably do this in a loop where i decrements  for ((i=99; i>=1; i--)) so you can just put $i instead of $((100-$i)). But the script works so well done!